intranet/templates/users/profile.html

```diff
@@ -257,11 +257,13 @@ <h3>
                             </th>
                             <td>
                                 {% for phone in profile_user.phones.all %}
+                                    {% if phone is None %}
```

Comment:
Are you sure this works? the logic seems backwards to me. You should make it `{% if phone %}`.